addons/point_of_sale/static/src/js/models.js

```diff
@@ -994,12 +1030,36 @@ export class PosGlobalState extends PosModel {
         this.synch = { status, pending };
     }
 
+    async pushUnpaidOrdersToServer(orders) {
```

Comment:
Do we really need it? Maybe if cross order? 
To check.

Reply:
For the moment this is obviously not necessary in the point_of_sale because the order button has been removed, orders are sent to preparation after payment validation, so the order does have a server_id. Nevertheless, we are still discussing the addition of the button.